WinUIGallery/ControlPages/ListViewPage.xaml

```diff
@@ -432,5 +432,130 @@ sent or received, and those values are bound in the DataTemplate.--&gt;
                 </x:String>
             </local:ControlExample.Xaml>
         </local:ControlExample>
+
+
+        <local:ControlExample HeaderText="ListView with context menus">
+            <StackPanel>
+                <TextBlock Margin="0 0 0 15" TextWrapping="Wrap" Text="This ListView allows users to right click to open a context menu. In this case, the context menu provides the option to delete an entry." />
+                <ListView
+                    x:Name="ContextMenuList"
+                    SelectionMode="Single"
+                    ShowsScrollingPlaceholders="True"
+                    Height="400"
+                    Width="400"
+                    BorderThickness="1"
+                    BorderBrush="{ThemeResource SystemControlForegroundBaseMediumLowBrush}"
```

Comment:
Nit: we shouldn't promote old brushes, but the ones that are listed in the Colors section instead